packages/eslint-config-react/index.js

```diff
@@ -1,15 +1,37 @@
+const error = 2;
+const warn = 1;
+const ignore = 0;
+
 module.exports = {
   extends: [
     "@socialgouv/eslint-config-recommended",
+    "airbnb",
```

Comment:
not react related

Reply:
Ok Wrong package total fail I’ll use Airbnb-react instead